src/main/cache/issuesCache.ts

```diff
@@ -72,10 +72,22 @@ export class IssuesCache {
      * @param workspace - the workspace to search it's data
      * @returns ScanResults if there are stored and relevant, undefined otherwise.
      */
-    public getOrClearIfNotRelevant(workspace: vscode.WorkspaceFolder): ScanResults | undefined {
+    public async getOrClearIfNotRelevant(workspace: vscode.WorkspaceFolder): Promise<ScanResults | undefined> {
```

Comment:
The term 'not relevant' is pretty vague.
Let's rename 'not relevant '-> 'not expired' everywhere in the code/comments